### PR TITLE
Remove misleading descriptions of subscriptions

### PR DESCRIPTION
**Pull request by arthuro555**
The subscriptions related messages have misleading descriptions, making sone free features sound paid. This makes GDevelop look like a freemium software when it isn't, and is just lying and wrong. This PR makes it clear what the subscriptions do exactly.

### CODE REVIEW

**Comment by Bouh** [comment]
The one-click export isn't locked behind a plan, this should be edited.
**Everything is free, the plans up the quantity of exports per days, and remove the messages.**
Nothing more.
Or I missed something because I used Pro plan too long.

File: newIDE/app/src/Profile/SubscriptionChecker.js [hunk 1]
```diff
@@ -128,9 +127,8 @@ export class SubscriptionCheckDialog extends React.Component<
             <Star style={styles.icon} />
             <Text style={styles.iconText}>
               <Trans>
-                Having a subscription allows you to use the one-click export for
-                Android, Windows, macOS and Linux, launch live previews over
-                wifi, disable the GDevelop splashscreen during loading and more!
+                Having a subscription allows you to use the one-click export and
```
Comment (Bouh) [inline newIDE/app/src/Profile/SubscriptionChecker.js:130]: One click export isn't locked by a plan,  there is 2 free export, just the user need an account with a free plan and is limited to 2 per day.

File: newIDE/app/src/Profile/SubscriptionChecker.js [hunk 2]
```diff
@@ -128,9 +127,8 @@ export class SubscriptionCheckDialog extends React.Component<
...
+                disable the GDevelop splashscreen during loading!
```
Comment (Bouh) [inline newIDE/app/src/Profile/SubscriptionChecker.js:131]: yes and we can also add that take a plan remove the message before some features.

**Comment by Bouh** [comment]
Maybe a table like this should be better, (non relevant mockup)
image
I've added edu plan, but there is no plan like that, it was for the example.

File: newIDE/app/src/Profile/SubscriptionChecker.js [hunk 3]
```diff
@@ -127,8 +127,9 @@ export class SubscriptionCheckDialog extends React.Component<
             <Star style={styles.icon} />
             <Text style={styles.iconText}>
               <Trans>
-                Having a subscription allows you to use the one-click export and
-                disable the GDevelop splashscreen during loading!
+                Get a subscription to gain more automatic exports and remove the
+                loading screens and this message asking you to get a
```
Comment (4ian) [inline newIDE/app/src/Profile/SubscriptionChecker.js:131]: "[...], remove the GDevelop splashscreen and [...]" I would say

**Review by Bouh — APPROVED** [review]

File: newIDE/app/src/Profile/SubscriptionChecker.js [hunk 4]
```diff
@@ -128,9 +127,9 @@ export class SubscriptionCheckDialog extends React.Component<
             <Star style={styles.icon} />
             <Text style={styles.iconText}>
               <Trans>
-                Having a subscription allows you to use the one-click export for
-                Android, Windows, macOS and Linux, launch live previews over
-                wifi, disable the GDevelop splashscreen during loading and more!
+                Get a subscription to gain more automatic exports and remove the
```
Comment (4ian) [inline newIDE/app/src/Profile/SubscriptionChecker.js:130]: Nit: I prefer one-click to automatic

File: newIDE/app/src/Profile/SubscriptionChecker.js [hunk 5]
```diff
@@ -128,9 +127,9 @@ export class SubscriptionCheckDialog extends React.Component<
...
+                GDevelop splashscreen and this message asking you to get a
```
Comment (4ian) [inline newIDE/app/src/Profile/SubscriptionChecker.js:131]: there are two "and" in this sentence, replace by a comma.

**Review by Bouh — APPROVED** [review]

File: newIDE/app/src/Profile/SubscriptionDialog.js [hunk 6]
```diff
@@ -188,11 +188,11 @@ export default class SubscriptionDialog extends React.Component<Props, State> {
...
                         <Text>
                           <Trans>
-                            Get a subscription to package your games for
-                            Android, Windows, macOS and Linux, use live preview
-                            over wifi and more. With a subscription, you're also
-                            supporting the development of GDevelop, which is an
-                            open-source software.
+                            Get a subscription to gain more automatic exports
+                            and remove the GDevelop splashscreen and messages
+                            asking you to get a subscription. With a
```
Comment (4ian) [inline newIDE/app/src/Profile/SubscriptionDialog.js:193]: This must be updated too ;)

**Pull request merged**